application/controllers/HistoryController.php

```diff
@@ -87,6 +88,10 @@ public function indexAction()
             Filter::like('service.id', '*')
         ));
 
+        if (OptimizerHints::shouldDisableOptimizerForHistoryQueries()) {
```

Comment:
any reason why this is not done by `disableOptimizer`? (It's only a temporary fix anyway, isn't it? So I don't care much about that `disableOptimizer` does not disable anything if disabled. :rofl: )

Reply:
Yeah, it's a temporary fix. I'm fine with combining the functions.